spec/models/asp/payment_request_spec.rb

```diff
@@ -32,13 +49,15 @@
     end
   end
 
-  describe "factory" do
+  describe "factories" do
     ASP::PaymentRequestStateMachine.states.each do |state|
       it "has a valid '#{state}' factory" do
         expect(create(:asp_payment_request, state)).to be_valid
       end
     end
 
+    # NOTE: a previous version of the factory was creating 2 records on each call
+    # The problem was solved using initialize_with which has different side-effects
```

Comment:
Can you explain which side effects are you talking about ? Otherwise I don't see the point in talking about the side effects.

Reply:
done